lib/internal/url.js

```diff
@@ -222,9 +227,7 @@ class URLSearchParams {
   }
 
   [inspect.custom](recurseTimes, ctx) {
-    if (!this || !this[searchParams] || this[searchParams][searchParams]) {
-      throw new ERR_INVALID_THIS('URLSearchParams');
-    }
```

Comment:
I would prefer something like...

```js
if (!isURLSearchParams(this))
  throw new ERR_INVALID_THIS('URLSearchParams');
```

keeping the throw outside of `isURLSearchParams()`